core/modules/proto/testProtocol.cc

```diff
@@ -61,9 +61,15 @@ struct ProtocolFixture : public lsst::qserv::proto::FakeProtocolFixture {
             && (t1.chunkid() == t2.chunkid())
             && (t1.db() == t2.db());
 
-        bool sTablesEq = t1.scantables_size() == t2.scantables_size();
-        for(int i=0; i < t1.scantables_size(); ++i) {
-            sTablesEq = sTablesEq && (t1.scantables(i) == t2.scantables(i));
+        bool sTablesEq = t1.scantable_size() == t2.scantable_size();
+        for(int i=0; i < t1.scantable_size(); ++i) {
+            auto const& sTbl1 = t1.scantable(i);
+            auto const& sTbl2 = t2.scantable(i);
+            bool eq = (sTbl1.db().compare(sTbl2.db()) == 0
+                       && sTbl1.table().compare(sTbl2.table()) == 0
```

Comment:
Is there a reason to prefer `sTbl1.table().compare(sTbl2.table()) == 0` over `sTbl1.table() == sTbl2.table()`?